graylog2-web-interface/src/components/common/ExpandableListItem.jsx

```diff
@@ -88,6 +88,7 @@ class ExpandableListItem extends React.Component {
   render() {
     const { expanded } = this.state;
     const { checked, expandable, selectable, header, subheader, children, ...otherProps } = this.props;
+    const headerToRender = selectable ? (<span role="button" tabIndex={0} onClick={() => { this._checkbox.click(); }}>{header}</span>) : header;
```

Comment:
I think we should move the callback function out of the render method. This [creates an additional function on every render](https://reactjs.org/docs/faq-functions.html#bind-in-render), and we will have to render and re-render many `ExpandableListItem` components.

Reply:
Good catch :+1: